### ISSUE TITLE
Add Dockerfile

### ISSUE THREAD

**Issue by dboreham** [cerc-io/laconic-registry-cli#3]
To allow users to run the CLI from a container, like:

```
$ docker run cerc/laconic-client:local --help
laconic <command>

Commands:
  laconic cns  CNS tools

Options:
      --version  Show version number                                   [boolean]
  -v, --verbose  Verbose output                       [boolean] [default: false]
  -c, --config   Config file path.              [string] [default: "config.yml"]
      --help     Show help                                             [boolean]
````

Prototyped here: https://github.com/cerc-io/stack-orchestrator/blob/main/container-build/cerc-laconic-client/Dockerfile

**Comment by ABastionOfSanity** [comment]
Working on this as part of github action testing/demo

**Comment by ABastionOfSanity** [comment]
@dboreham Should this just use:
https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/container-build/cerc-laconic-registry-cli/Dockerfile#L55
as mentioned here:
https://github.com/cerc-io/laconic-registry-cli/issues/15#issuecomment-1379125090

**Comment by dboreham** [comment]
Yes, I think this issue report pre-dates S-O, and certainly the Dockerfile there. 
But, ideally I think the Dockerfile should be in _this_ repo. That's the model we use where possible.
The presence of a Dockerfile in the S-O repo indicates that there wasn't one, or a usable one in the target project.
So tentatively I'd move the Dockerfile here and we can update S-O to use it vs having its own one.

**Comment by zramsay** [comment]
closed in #21
